frontend/src/components/navbar/navstyle.module.scss

```
@@ -0,0 +1,38 @@
+.toolBar{
+    position:fixed;
+    width: 100%;
+    height: 56px;
+    background: transparent;
+    top:0;
+    left:0;
+    z-index: 1;
+}
+.toolBarNav{
+    display:flex;
+    align-items: center;
+    height: 100%;
+    padding:0 1rem;
+}
+.toolBarItems ul{
+    list-style: none;
+    // margin:0;
+    // padding:0;
+    display:flex;
+}
+.toolBarRef{
+    text-decoration: none;
+    // padding:20px 0.5rem 20px 0.5rem;
+    padding:20px 10px 20px 10px;
+    color: rgb(255, 166, 0);
+}
+.toolBarRefActive{
+    border-bottom: 1px solid rgb(255, 166, 0);
+    //border-image: linear-gradient(to top, rgb(255, 166, 0) 50%);
+    border-image-slice: 1;
+}
+.spacer{
+    flex: 1;
+}
```

Tené en cuenta que es más legible tener las clases separadas por líneas vacías:
```suggestion
}

.toolBarLogo {
```